### DISCUSSION TITLE
Realtime should default to last 30 minute views

### DISCUSSION THREAD

**Discussion by andrewholgate** [plausible/analytics#1532] [Anything Else]
When viewing the realtime user activity, my expectation would be for all the panels on the dashboard to display the last 30 minute values and not the "Current users" values. This is what Google Analytics does in the realtime view and I'm sure other users of Plausible would likely expect this behaviour as well.

Interestingly, the Goal Conversations defaults to 30 minutes (which I find very useful) however the rest are all current users. This also creates an inconsistent view of the data in realtime

**Comment by Olakeen** [comment]
Up this, in this state, it is useless
![2022-06-16 180331](https://user-images.githubusercontent.com/26546237/174115427-47840ced-ef84-42b5-aaf3-a06151d8cff1.jpg)

**Reply by metmarkosaric** [reply]
thanks for the feedback @Olakeen! seems like the needs for this vary from site to site. we've recently onboarded several large sites coming from GA and they all looked specifically for max 5 mins for the realtime dashboard. 30 mins was too long for them. perhaps in the future we can introduce an option so you can select the default time limit that you prefer.

**Reply by Olakeen** [reply]
> perhaps in the future we can introduce an option so you can select the default time limit that you prefer.

Yes it will be great 👍

**Reply by andrewholgate** [reply]
The issue is that "Real time" is confusing as it shows a mixture of users over the last 5 minutes and 30 minutes.

Why not just create 2 different options? "Real time" for users over the past 5 minutes and a second options for "30 minutes"?

**Reply by metmarkosaric** [reply]
yeah, ideally we'll give you an option to choose. only 3 metrics are last 30 mins and we mark them with the note "last 30 min": unique visitors and pageviews in the top chart and the goals section. the rest of the realtime dashboard is all "last 5 mins".
